cip-TOKEN-STD/cip-meiersi-Canton-Network-Token-Standard.md

```diff
@@ -0,0 +1,574 @@
+There is a one-to-one correspondence between Daml contracts and UTXOs.
+Canton's UTXOs are annotated with their stakeholders and are only distributed to the nodes hosting these stakeholders.
+
+Constructing transactions requires access to all UTXOs referenced or consumed by the transaction.
+Clients provide this access by retrieving the UTXOs known to their parties from their validator node and
+the UTXOs known to an app provider using off-ledger API calls to app-specific services.
+
+The standard proposes to provide UTXO access for constructing transactions involving tokens as follows:
+
+- **wallet access to user parties**:
+  wallets are assumed to have access to the Ledger API of the validator node hosting the parties of their users
+  and use that to retrieve all UTXOs known to these parties.
+- **registry off-ledger APIs**:
+  registries serve UTXOs private to the registry via standardized HTTP APIs specified using OpenAPI as part of the standard.
+
+
+#### Direct Peer-to-Peer / Free of Payment (FOP) Transfer Workflow
+
+The FOP transfer workflow supported by the standard enables an investor to send a
+specific amount of their asset holdings to another investor, which is considered
+the recipient of the transfer.
+
+The transfer is always initiated by the sender using their wallet to submit a
+Daml transaction that specifies the instruction to the registry to execute the
+transfer within a given deadline. Depending on the registry, this instruction
+gets completed immediately as part of this one Daml transaction; or once
+further registry-specific steps have happened as part of additional Daml transactions.
+
+The latter option is provided for registries that require additional approvals
+to be provided before a transfer can be executed; or registries whose authoritative
+ledger is maintained outside of Canton. Registries are allowed to abort a
+transfer instruction in case the transfer is not possible.
+
+
+#### Delivery versus Payment (DVP) Transfer Workflows
+
+The DVP transfer workflows enable multiple asset transfers to be executed as
+part of a single Daml transaction in an all-or-nothing fashion: either all
+transfers happen or none of them happens.
+
+For this purpose, the registries allow investors to use their wallet to allocate
+some of their asset holdings to a settlement for a fixed amount of time. Once
+all allocations required for a settlement are present, the app executing the settlement
+submits the one Daml transaction that triggers all of the transfers.
+
+More concretely, the standard specifies APIs that enable apps and wallets to
+support the workflows along the following lines:
+
+1. The user uses the app to get to a point where they are required to deliver
+   some of their assets as part of a settlement. For example, they made a
+   matched bid on an exchange app or they requested to purchase a good in
+   exchange for some of their asset holdings.
```

Comment:
I think we use
- assets
- asset holdings
- tokens

to describe the same thing. Not sure which term resonates best.

Reply:
fixed: removed token holdings and prefer asset holdings, refer to "asset tokenization" to connect assets to tokens.